### PR TITLE
Structure

### PR DESCRIPTION
**Pull request by AlaaAlsalem**
# Here are the things done for this project
> Install the webpack 
> Create the src folder 
> Add the index.html file to hold the structure of the project
> Add the index.js to hold the logic of the project
> Add style.css to style the project

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "rto_do_list",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```
Comment (omar-labana) [inline package.json]: - [ ] Kindly update your npm scripts to add basic commands for building Webpack in development mode, and webpack dev server. it should be something like this:
```JSON
"scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "start": "webpack serve --open",
    "build": "webpack"
  }
```

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    <div class="task">
+    <div class="wrapper">
+        <input type="checkbox" name="" class="box m-input" data-id=${item.index} ${item.completed ? 'checked' : ''}>
+        <input type="text" name="" data-id="${item.index}" value="${item.description}">
+    </div>
+        <i id="ellipse" class="fas fa-ellipsis-v m-i"></i>
+        
+    </div>
+    <hr>`;
+    container.innerHTML += containerItems;
```
Comment (omar-labana) [inline src/index.js:42]: - [ ] _[OPTIONA]_ I'd suggest increasing the `font-size` and padding around each task to make it easier to read.
- [ ] _[OPTIONA]_ I'd suggest using a `ul` element then add each task as a list item or if you prefer another semantic approach.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @AlaaAlsalem , 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @AlaaAlsalem , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

**Pull request merged**